src/wasm-interpreter.h

```diff
@@ -240,7 +240,7 @@ class ExpressionRunner : public Visitor<SubType, Flow> {
         case ConvertUInt32ToFloat64: return value.convertUToF64();
         case ConvertSInt32ToFloat32: return value.convertSToF32();
         case ConvertSInt32ToFloat64: return value.convertSToF64();
-        default: abort();
+        default: WASM_UNREACHABLE();
```

Comment:
I think if the intent here is to make it more robust against/not crash with unexpected input then this should be a throw or some error return, or something that has consistent behavior in release builds.  `WASM_UNREACHABLE` is just `__builtin_unreachable` in that case and it means the compiler will just optimize assuming it never happens and probably crash somewhere else.

Or is this intended to indicate that we are now confident (because of the parser hardening) that this will actually never happen? In that case then I agree that unreachable makes sense.